src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1347,12 +1498,14 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             bool disableNotification = default,
             int replyToMessageId = default,
             InlineKeyboardMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.